src/nccl_ofi_scheduler.c

```diff
@@ -22,10 +22,25 @@ static inline size_t sizeof_schedule(int num_rails)
 		+ num_rails * sizeof(nccl_net_ofi_xfer_info_t);
 }
 
-void nccl_net_ofi_set_multiplexing_schedule(size_t size, int num_rails,
-					    size_t align,
-					    nccl_net_ofi_schedule_t *schedule)
+void nccl_net_ofi_set_multiplexed_round_robin_schedule(nccl_net_ofi_threshold_scheduler_t *scheduler,
+												   size_t size,
+												   int num_rails,
+												   int num_stripes,
+												   size_t align,
+												   nccl_net_ofi_schedule_t *schedule)
 {
+	if (OFI_UNLIKELY(num_rails == 0)) return;
+
+	/* TODO Number of stripes cannot be greater than number of rails */
```

Comment:
Don't really understand how this can be a TODO?

Reply:
Fixed it. Added the check for num_stripes to be not greater than num_rails.